Makefile

```diff
@@ -63,14 +63,13 @@ test-dependencies:
 	@pip install -q -r test-requirements.txt
 
 test: test-dependencies test-start-minio ## run tests quickly with the default Python
-	coverage run -m pytest -v || $$(make test-stop-minio)
-	@make test-stop-minio
+	coverage run -m pytest; r=$$?; $(MAKE) test-stop-minio; exit $$r
```

Comment:
Could you please add `-v` back on the command line?  I find this helpful during development.

Reply:
Yes, sorry. I didn't remove it on purpose.

Reply:
No worries - thanks for the update.